animareflection/ui
# Slider Component

## Description

##### Task link: https://trello.com/c/lwzboTSK/120-slider-component

Added `Slider` component, recipe, primitives, stories, and tests.

## Test Steps

1) Verify that test structure is appropriate and that all tests pass.
2) Verify that component renders appropriately in both storybook and a downstream app
3) Verify that the styles are responsive when theme is switched.
4) Verify that styles look good.
